assets/js/search.js

```diff
@@ -39,62 +39,66 @@
   /**
    * Function displays the total items displayed on the page
    */
-  function showItemsCount(searchEntry, totalCount, numSubmodules) {
-    if (searchEntry.type === 'libraryEntries') {
+  function showItemsCount(totalCount, numSubmodules) {
+    if ($('.table-clickable-row'.length > 0)) {
       $('#search-results-count').show();
       if (totalCount > 0 && numSubmodules > 0) {
-        $('#search-results-count').html("<strong>" + totalCount + "</strong> repos (~<strong>" + numSubmodules + "</strong> modules)");
+        $('#search-results-count').html("<strong>" + totalCount + "</strong>repos(~<strong>" + numSubmodules + "</strong> modules)");
       } else {
         $('#search-results-count').text("0 repos");
       }
-    } else {
-      if (totalCount > 0 ) {
-        $('#search-results-count').html("<strong>" + totalCount + "</strong> post(s) found");
-      }
     }
   }
 
   /**
    * Function to display all items on the page
    */
   function showAllItems() {
-    $('.guide-card').show();
-    $('.category-head').show();
-    $('.categories ul li').show();
+    return (
+      $('.guide-card').show() &&
+      $('.category-head').show() &&
+      $('.categories ul li').show()
```

Comment:
Why change to `&&` here and below?

Reply:
I felt using `&&` was better since I was returning. Is it better to terminate each statement instead?